setup.cfg

```diff
@@ -1,19 +1,28 @@
 [flake8]
-min-version=2.7
-accept-encodings=utf-8
-# Errors we ignore:
-# - E121,E123,E126,E24,E704,W503,W504 flake8 default ignores, excluding E226 (have to be listed here to not be overridden)
-# - E221: multiple spaces before operator (used to align visually)
-# - E731: do not assign a lambda expression, use a def
-# - F405 object may be undefined, or defined from star imports
-# - C901: function/method complexity
-# - E305: spacing after a declaration (might be nice to change eventually)
-# `flake8-future-import` errors we ignore:
-# - FI50: `__future__` import "division" present
-# - FI51: `__future__` import "absolute_import" present
-# - FI12: `__future__` import "with_statement" missing
-# - FI53: `__future__` import "print_function" present
-# - FI14: `__future__` import "unicode_literals" missing
-# - FI15: `__future__` import "generator_stop" missing
-# - E741: ambiguous variable name
-ignore=E121,E123,E126,E24,E704,W503,W504,E305,C901,E221,E731,F405,FI50,FI51,FI12,FI53,FI14,FI15,E741
+min-version = 2.7
+accept-encodings = utf-8
+extend-exclude = test
```

Comment:
I don't think this would be good to merge in, as it would affect all linting for test files. I think these changes would have to be all or nothing for each error.

Reply:
Opening PR to start discussion.

I can get tests done tonight if no objections.